### PR TITLE
Added instructions for getting PinPog running on MacOS.

### PR DESCRIPTION
**Pull request by amirrajan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,3 +21,42 @@ $ make
...
+The following instructions have only been tested on Mojave 10.14.5
+using Xcode 10.2.
+
+### Setting up your Mac as a dev machine
+
+Your life will be overall easier if you do the following. You only
+have to do these steps once on your Mac to make it a "developer
+machine".
+
+1. Install Xcode 10.2 from the AppStore. If you don't want to install
```
Comment (rexim) [inline README.md]: Why do we need Xcode?

File: README.md [hunk 2]
```diff
@@ -21,3 +21,42 @@ $ make
...
+   from the AppStore, you can download it from
+   http://developer.apple.com/more/ .
+2. Open up the terminal and run `sudo xcode-select --install`
+3. Install brew from http://brew.sh .
+3. `brew install gcc`
+4. `brew install cmake`
```
Comment (rexim) [inline README.md]: Why do we need cmake?

File: README.md [hunk 3]
```diff
@@ -21,3 +21,42 @@ $ make
...
+
+1. `make`
+
+You will see the following warning, but it's fine.
+
+```
+WARNING: Image format was not specified for 'pinpog' and probing guessed raw.
+         Automatically detecting the format is dangerous for raw images, write operations on block 0 will be restricted.
+         Specify the 'raw' format explicitly to remove the restrictions.
+```
```
Comment (rexim) [inline README.md]: The warning exists on Linux too. Should be a separate issue. (which is extremely easy to fix btw)

File: README.md [hunk 4]
```diff
@@ -21,3 +21,42 @@ $ make
...
+Your life will be overall easier if you do the following. You only
+have to do these steps once on your Mac to make it a "developer
+machine".
+
+1. Install Xcode 10.2 from the AppStore. If you don't want to install
+   from the AppStore, you can download it from
+   http://developer.apple.com/more/ .
+2. Open up the terminal and run `sudo xcode-select --install`
+3. Install brew from http://brew.sh .
+3. `brew install gcc`
```
Comment (rexim) [inline README.md]: Why do we need gcc?

**Review by rexim — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (amirrajan) [reply README.md]: Will update PR to remove the "novice" help guide.

File: README.md (hunk 2 above)
Comment (amirrajan) [reply README.md]: I confirmed that they are not needed if you have `brew`.

File: README.md (hunk 3 above)
Comment (amirrajan) [reply README.md]: Will remove this.

File: README.md (hunk 4 above)
Comment (amirrajan) [reply README.md]: Not needed (confirmed).

File: README.md [hunk 5]
```diff
@@ -21,3 +21,15 @@ $ make
 
 [nasm]: https://www.nasm.us/
 [qemu]: https://www.qemu.org/
```
Comment (rexim) [inline README.md:23]: Please keep the link labels at the bottom of the document

File: README.md [hunk 6]
```diff
@@ -21,3 +21,15 @@ $ make
...
+
+## Building on MacOS
```
Comment (rexim) [inline README.md:25]: Please make this a subsection of the "Dependencies" section. It's about installing dependencies anyway.

File: README.md [hunk 7]
```diff
@@ -21,3 +21,15 @@ $ make
...
+
+1. `brew install nasm`
+2. `brew install qemu`
```
Comment (rexim) [inline README.md:28]: AFAIK, `brew` can install multiple packages at once. Let's squash this into a single line like `brew install nasm qemu`, so it's easy to copy-paste.

File: README.md [hunk 8]
```diff
@@ -21,3 +21,15 @@ $ make
...
+
+### Running
+
+1. `make`
```
Comment (rexim) [inline README.md:32]: Completely redundant information. We already have that in "Quick Start" section.

**Review by rexim — CHANGES_REQUESTED** [review]

**Comment by amirrajan** [comment]
Closing the PR. The existing instructions are straight forward assuming that an experienced dev on Mac knows how to use `brew`.  Both `nasm` and `qemu` are available in the package manager. 

I generally assume that I’m working with complete novices, which is why the initial PR went into so much detail about getting a dev environment created. Based on the stream from yesterday, it’s clear that this extra information isn’t valued for this project (which is totally fine and respected). 

Thanks for taking the time to look at he PR!

**Pull request closed without merging**